src/plugins/files/server/routes/common.ts

```diff
@@ -20,7 +21,7 @@ export function getDownloadHeadersForFile({ file, fileName }: Args): ResponseHea
     'content-type':
       (fileName && mime.getType(fileName)) ?? file.data.mimeType ?? 'application/octet-stream',
     // Note, this name can be overridden by the client if set via a "download" attribute on the HTML tag.
-    'content-disposition': `attachment; filename="${fileName || getDownloadedFileName(file)}"`,
+    'content-disposition': contentDisposition(fileName || getDownloadedFileName(file)),
```

Comment:
I expected this fix to have us use `res.file` instead of `res.ok` in the areas where file downloads are given in request contexts.

https://github.com/elastic/kibana/blob/939f3dfc48419ef2fab8303adbb5c7a68c2bc0c6/src/plugins/files/server/routes/public_facing/download.ts#L47-L50
https://github.com/elastic/kibana/blob/939f3dfc48419ef2fab8303adbb5c7a68c2bc0c6/src/plugins/files/server/routes/file_kind/download.ts#L42-L45

Would using `res.file` work as a simpler change to fix this?

Reply:
it actually is... thanks for the suggestion!